docs/design.md

```diff
@@ -281,12 +281,12 @@ a NOOP.
 ***Transaction abort:***
 
 This is the case in which a transaction, upon reading its transaction
-table entry, finds that it has been aborted. In this case, the
-transaction can not reuse its intents; it returns control to the client
-before cleaning them up (other readers and writers would clean up
-dangling intents as they encounter them) but will make an effort to
-clean up after itself. The next attempt (if applicable) then runs as a
-new transaction with **a new tx id**.
+record, finds that it has been aborted. In this case, the transaction
+can not reuse its intents; it returns control to the client before
```

Comment:
It never really reuses intents. The intents' primary role is that of preventing starvation by forcing conflicts.